src/EditorFeatures/Test/Diagnostics/DiagnosticAnalyzerServiceTests.cs

```diff
@@ -69,7 +69,8 @@ public async Task TestHasSuccessfullyLoadedBeingFalse()
 
         var diagnostics = await analyzer.GetDiagnosticsForIdsAsync(
             workspace.CurrentSolution, projectId: null, documentId: null, diagnosticIds: null, shouldIncludeAnalyzer: null, getDocuments: null,
-            includeSuppressedDiagnostics: false, includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: false, CancellationToken.None);
+            includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: false, CancellationToken.None);
+        diagnostics = diagnostics.WhereAsArray(d => !d.IsSuppressed);
```

Comment:
note: code that passes `includeSuppressedDiagnostics: false` needs to filter the results.  I do that for features.  HOwever, for tests, it doesn't seem to have any impact on results.  so for simplicity i'm not doing it.